ts/session/messages/outgoing/visibleMessage/OpenGroupVisibleMessage.ts

```diff
@@ -1,3 +1,25 @@
-import { VisibleMessage } from './VisibleMessage';
+import { SettingsKey } from '../../../../data/settings-key';
+import { SignalService } from '../../../../protobuf';
+import { Storage } from '../../../../util/storage';
+import { VisibleMessage, VisibleMessageParams } from './VisibleMessage';
 
-export class OpenGroupVisibleMessage extends VisibleMessage {}
+// eslint-disable-next-line @typescript-eslint/ban-types
+export type OpenGroupVisibleMessageParams = VisibleMessageParams & {};
+
+export class OpenGroupVisibleMessage extends VisibleMessage {
+  private readonly blocksCommunityMessageRequests: boolean;
+
+  constructor(params: OpenGroupVisibleMessageParams) {
+    super(params);
+    // they are the opposite of each others
+    this.blocksCommunityMessageRequests = !Storage.get(SettingsKey.hasBlindedMsgRequestsSetting);
```

Comment:
This answers my question above and so can we rename this setting to be more explicit like `hasBlindedMsgRequestsSettingEnabled` or something similar please? 🙏

Reply:
renamed to hasBlindedMsgRequestsEnabled